internal/testing/e2e/in_memory.go

```diff
@@ -123,3 +146,32 @@ func createMinimatchForTest(t *testing.T) *rpcTesting.TestContext {
 	tc.AddCloseFunc(closer)
 	return tc
 }
+
+// Create a mmf service using a started test server.
+// Inject the port config of mmlogic using that the passed in test server
+func createMatchFunctionForTest(t *testing.T, c *rpcTesting.TestContext) *rpcTesting.TestContext {
```

Comment:
does this render the setup files in current e2e minimatch tests to setup mmf, evaluator obsolete? Are we not removing them for now because there are other tests using them that need to be ported?

Reply:
My plan is to migrate the setup from `test/e2e` folder to `internal/testing/e2e` folder since it abstract the implementation details. Starting from this PR, all the end to end tests will use the setup code in this file. I'll obsolete the original setup codes in a later PR.